aep/general/0134/aep.md.j2

```diff
@@ -27,34 +27,30 @@ Update methods are specified using the following pattern:
   - If the update RPC is [long-running](#long-running-update), the response
     **must** be an `Operation` for which the return type is the resource
     itself.
-- The method **should** support partial resource update, and the HTTP verb
-  **should** be `PATCH`.
+- The method **should** support partial resource update,
```

Comment:
We should make this a must.

[HTTP Patch semantics](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH) say PATCH is for partial updates. If we mandate PATCH, we should mandate partial resource update as well.

Reply:
yeah it's a good point.... that might be it's own can of worms so I'll make that a follow-up PR.